plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/JfrogArtifactoryRepository.tsx

```diff
@@ -80,11 +82,8 @@ export function JfrogArtifactoryRepository(props: RepositoryProps) {
   );
 }
 
-JfrogArtifactoryRepository.defaultProps = {
-  title: 'Docker Images',
-};
 interface RepositoryProps {
-  widget: boolean;
-  image: string;
-  title: string;
+  widget?: boolean;
+  image?: string;
```

Comment:
Will this is optional?

Reply:
I make the three interface values optional because I only need to pass title as default value.
in this format:
```
export function JfrogArtifactoryRepository(
  props: RepositoryProps = { title: 'Docker Images' },
) {
...
```

if i don't make those 3 optional it was causing some placeholder error as the three value of the interface needs to be passed with in {}. 
But i think my approach of making all interface properties optional is not right.

Reply:
follow this https://github.com/janus-idp/backstage-plugins/pull/649/files#r1302454640 and this change will not be needed so keep it as required



```
export function JfrogArtifactoryRepository(
  props: RepositoryProps = { title: 'Docker Images' },
) {
...
```

moreover, the above won't work as you are expecting it to, as if props will be defined then `{ title: 'Docker Images' }` will never be assigned. But what we want is props should pick the image and widget from what's coming as prop and if the title is undefined then pass the default value.